src/containers/SubjectViewer.jsx

```diff
@@ -193,7 +192,7 @@ class SubjectViewer extends React.Component {
     //Make sure we monitor visible size of Subject Viewer.
     window.addEventListener('resize', this.updateSize);
     this.updateSize();
-    this.props.dispatch(fetchSubject());  //Fetch the first subject.
+    this.props.dispatch(fetchSubject());  //Fetch the first subject. This will also ensure a clean slate for Annotations, Previous Annotations, and Classifications.
```

Comment:
This dispatch call now clears annotations made on the classifier when navigating to another location on the app and returning to the classifier (ex. classifier -> About -> classifier erases work that was done).